### PR TITLE
[#1207] remove common-lang3 and update qulice

### PR DESCRIPTION
**Pull request by pnatashap**
- remove dependency to commons-lang3
- update qulice to the latest

includes #1915

### CODE REVIEW

**Comment by pnatashap** [comment]
@yegor256 please take a look

File: src/main/java/com/rultor/web/TkAppFallback.java [hunk 1]
```diff
@@ -105,7 +104,7 @@ private static Response fatal(final RqFallback req) throws IOException {
                     TkAppFallback.class.getResource("error.html.vm"),
                     new RsVelocity.Pair(
                         "err",
-                        ExceptionUtils.getStackTrace(req.throwable())
+                        req.throwable().getStackTrace()
```
Comment (yegor256) [inline src/main/java/com/rultor/web/TkAppFallback.java:107]: @pnatashap I believe, this is not exactly what `ExceptionUtils.getStackTrace` does. We should use `Logger.format("%[exception]s", req.throwable())` instead.

**Comment by yegor256** [comment]
@pnatashap looks great! one small comment above

**Comment by pnatashap** [comment]
@yegor256 fixed

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@rultor merge

**Comment by yegor256** [comment]
@pnatashap thanks a lot!

**Pull request merged**